pecos/xmc/xtransformer/network.py

```diff
@@ -328,6 +333,74 @@ def forward(
         }
 
 
+@add_start_docstrings(
+    """XLM-Roberta Model with mutli-label classification head on top for XMC.\n""",
+    XLM_ROBERTA_START_DOCSTRING,
+)
+class XLMRobertaForXMC(BertPreTrainedModel):
```

Comment:
Instead of copying the implementaions from above, I'd suggest to reuse the code in `RobertaForXMC ` and override the `config_class` only. This is also used in huggingface implementation for `xlm_roberta`: [code](https://github.com/huggingface/transformers/blob/master/src/transformers/models/xlm_roberta/modeling_xlm_roberta.py#L67).

Reply:
That being said, the class `XLMRobertaForXMC` can be written as
```
@add_start_docstrings(...)
class XLMRobertaForXMC(RobertaForXMC):
    """
    some docstring
    """
    config_class = XLMRobertaConfig
```